requirements.txt

```diff
@@ -1,3 +1,2 @@
-numpy
-pandas
-anytree
\ No newline at end of file
+__future__
+qqdm
```

Comment:
qqdm requires: 
addict
juypter

If all we need is a good console progress bar, I personally have used Rich and it's pretty nice and doesn't require anything else. Not sure if there is a performanc hit on this either